gulpfile.js/tasks/style.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+const path = require('path')
+const gulp = require('gulp')
+const sass = require('gulp-sass')
+const config = require('../config')
+const sourceMaps = require('gulp-sourcemaps')
+const autoprefixer = require('gulp-autoprefixer')
+const plumber = require('gulp-plumber')
+const replace = require('gulp-replace')
+const gutil = require('gulp-util')
+const rename = require('gulp-rename')
+
+function sassToCss (stream) {
+  return stream
+    .pipe(sourceMaps.init())
+    .pipe(plumber((error) => {
+      gutil.log(gutil.colors.red(error.message))
+      this.emit('end')
+    }))
+    .pipe(sass(config.sass.settings))
+    .pipe(autoprefixer({browsers: ['last 2 versions', 'IE >= 8']}))
+    .pipe(rename({suffix: '.min'}))
+    .pipe(sourceMaps.write(config.sass.sourceMapsDir))
+    .pipe(gulp.dest(path.join(config.dest[gutil.env.version], config.sass.destDirName)))
+}
+
+gulp.task('style', ['stylelint'], () => {
```

Comment:
I'm not sure `v3` being the "default" style task is the right thing to do. Maybe we should have the `style` task run both a `style:v3` and the `style:v4` tasks 🤔.